SlackBotPrototype/API/SqLitePersistence.cs

```diff
@@ -0,0 +1,84 @@
+﻿using System;
+using System.Linq;
+using SQLite;
+
+namespace API
+{
+	// SHORTCUT: Normally we'd break out the repositories into
+	// a file per entity. Keep them together for simplicity here.
+	// 
+	// Also skipping tests for these because we'd need to set up integration tests
+	// with a SQLite database. If we weren't using a embedded file DB like SQLite, testing

```

Comment:
This is not a comprehensive list, but what comes top of mind for me is:

* Ensuring accurate state of DB between individual tests and test runs
  * Could reset DB between each run
  * Or create a new DB for each test run
* Depending on data required for tests, size of DB could be a factor especially if we go one DB/test -- would affect test speed, build times, deploy times
* Would have to properly clean up DBs artifacts from tests otherwise your build server won't be happy